scripts/qiita-auto-processing

```diff
@@ -194,23 +217,33 @@ for pipeline in full_pipelines:
     #          of Step 4 but for debugging it makes sense to separate
     artifact_process = []
     children_compliant = []
+    cmd = commands[0]
     for a in artifacts_compliant:
-        cmd = commands[0]
         # getting all jobs, includen hiddens, in case the job failed
         jobs = a.jobs(cmd=cmd['command'], show_hidden=True)
-        params = [j.parameters.values for j in jobs]
+        params = [{k: str(v) for k, v in j.parameters.values.items()
```

Comment:
Why is the string conversion needed here?

Reply:
Excellent question; for some reason (for old jobs) the parameters are not string and the parameters here are defined as strings; the easiest is to make sure that everything is string